.github/workflows/ci.yml

```diff
@@ -171,7 +171,7 @@ jobs:
         options: --health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=10s --health-timeout=5s --health-retries=10
 
     env:
-      artifacts_name: ${{ matrix.name }}
+      artifacts_name: ${{ matrix.artifacts_name }}
```

Comment:
What's this change about? Doesn't seem related to what you've mentioned in the PR description.

Reply:
Reverted